amr-wind/overset/TiogaInterface.cpp

```diff
@@ -102,20 +104,45 @@ void TiogaInterface::post_regrid_actions()
 
 void TiogaInterface::pre_overset_conn_work()
 {
+
+    auto& repo = m_sim.repo();
+    const int num_ghost = m_sim.pde_manager().num_ghost_state();
+    m_iblank_cell_host = repo.create_int_scratch_field_on_host("iblank_cell_host",1,num_ghost, FieldLoc::CELL);
+    m_iblank_node_host = repo.create_int_scratch_field_on_host("iblank_node_host",1,num_ghost, FieldLoc::NODE);
+
+    amr_to_tioga_iblank();
+
     m_iblank_cell.setVal(1);
     m_iblank_node.setVal(1);
+
+    (*m_iblank_cell_host).setVal(1);
```

Comment:
can just do `m_iblank_cell_host->setVal` no?

Reply:
Calling setVal here requires the dereferencing of a pointer since m_iblank_cell_host and m_iblank_node_host are unique_ptrs to an IntScratchField, while m_iblank_cell and m_iblank_node are references to IntField objects.